apps/fabric-website/src/components/App/App.tsx

```diff
@@ -107,4 +118,120 @@ export class App extends React.Component<IAppProps, any> {
     }
     return height;
   }
+
+  // Gets the section title by comparing the first part of the current URL to
```

Comment:
Nit: these really should just use jsdoc format so you actually benefit from your effort of writing comments :)

Reply:
Sounds reasonable to me!